pwndbg/ui.py

```diff
@@ -67,9 +69,27 @@ def get_window_size(target=sys.stdin):
     fallback = (int(os.environ.get("LINES", 20)), int(os.environ.get("COLUMNS", 80)))
     if not target.isatty():
         return fallback
+    rows, cols = get_cmd_window_size()
+    if rows is not None and cols is not None:
+        return rows, cols
     try:
         # get terminal size and force ret buffer len of 4 bytes for safe unpacking by passing equally long arg
         rows, cols = struct.unpack("hh", fcntl.ioctl(target.fileno(), termios.TIOCGWINSZ, b"1234"))
     except Exception:
         rows, cols = fallback
     return rows, cols
+
+
+def get_cmd_window_size():
+    info_out = gdb.execute("info win", to_string=True).split()
```

Comment:
Can you please add a docstring with all possible outputs of `info win`?

Also, I am pretty sure this is still far from ideal considering GDB translations :(, but maybe we can't deal with that anyhow right now.

Reply:
Ping @ThomsonTan